packages/strapi/lib/core/app-configuration/env-helper.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const _ = require('lodash');
+
+function env(key, defaultValue = undefined) {
+  return _.has(process.env, key) ? process.env[key] : defaultValue;
+}
+
+const utils = {
+  int(...args) {
+    const value = env(...args);
+    return typeof value === 'undefined' ? value : parseInt(value, 10);
+  },
+
+  bool(...args) {
+    const value = env(...args);
+    return value === 'true';
```

Comment:
if the default value is `true` (not a string), it would return false, is it a feature or a bug?

Reply:
Bug :) gonna fix this